### PR TITLE
Release 4.2.0

### PR DESCRIPTION
**Pull request by aweibel**

### CODE REVIEW

**Comment by aweibel** [comment]
@brandonferrua please check out the releasenotes.md, readme.md and package.json. No need to check the binary files.

@sarahmei I removed the contributions.md and added a note at the end of the readme.md if you wouldn't mind taking a quick look.

File: README.md [hunk 1]
```diff
@@ -24,66 +24,56 @@ Download the most recent version of [Sketch](https://www.sketchapp.com/).
...
 ![The tab symbol uses nested symbols for Hover, Default and Selected states](https://user-images.githubusercontent.com/1750832/41117592-58941256-6a42-11e8-9be7-d9a8bbe9b339.png)
 <br />
 <br />
 
-**Color Token Symbols**
+**Color Token and Typography Layer Styles**
 
-Salesforce Lightning Design System uses design tokens as named entities that store visual design attributes. The SLDS Component Library document uses color symbols to mimic [Lightning Design System design tokens](https://www.lightningdesignsystem.com/design-tokens/) with color values. This allows the structure of a symbol to use the equivalent of a design token.
+Salesforce Lightning Design System uses design tokens as named entities that store visual design attributes. The SLDS Component Library uses Sketch Layer Styles and Sketch Text Styles to mimic [Lightning Design System design tokens](https://www.lightningdesignsystem.com/design-tokens/). This allows Sketch designs to use the equivalent of a design token--saving you time spec'ing your designs for development.
```
Comment (brandonferrua) [inline README.md]: `design token; saving you...`

File: README.md [hunk 2]
```diff
@@ -104,35 +94,15 @@ In symbols, like icons, that contain solid color backgrounds, you’ll find the
...
+## Contributing, Feature Requests and Bug Reporting
 
-**Runner**
+The SLDS team welcomes your help in maintaining the SLDS Component Library. Please add any bugs or feature requests under the [Issues](https://github.com/salesforce-ux/design-system-ui-kit/issues) tab of this repository.
 
-Sketch Runner helps you to get around Sketch quicker by giving you an intuitive interface to run commands directly from your keyboard. Runner allows you to skip Sketch navigation, buttons or menus and quickly type the action you want to take like, “insert desktop tab” to add a tab component symbol to your artboard.
-[https://sketchrunner.com](https://sketchrunner.com/)
+**Public contributions are currently closed.** Salesforce designers contribute to this kit while designing Salesforce releases through an application called Abstract. These releases are completed four months before being released to the public, therefore public contributions to this kit are essentially contributing to a four month old stale release. This become unmanagable to merge Sketch binary files and isn't fair to contributors who work hard on a contribution only to have it closed without a merge.
```
Comment (brandonferrua) [inline README.md]: You may want this callout at the top as well

🚨 **External contributions are currently closed.** 🚨 

> Salesforce designers contribute to this kit while designing Salesforce releases through an application called Abstract. These releases are completed four months before being released to the public, therefore public contributions to this kit are essentially contributing to a four month old stale release. This become unmanagable to merge Sketch binary files and isn't fair to contributors who work hard on a contribution only to have it closed without a merge.

File: README.md [hunk 3]
```diff
@@ -24,66 +24,56 @@ Download the most recent version of [Sketch](https://www.sketchapp.com/).
 
 ### Salesforce Sans fonts
 
-1. [Download the Design System Zip](https://www.lightningdesignsystem.com/resources/downloads/)
-2. Find the fonts located in the `/assets/fonts` directory
-3. Install the fonts on your system
+1. Download the [Salesforce Sans fonts](https://github.com/salesforce-ux/design-system/tree/master/assets/fonts) from the Design System repository
+2. Install the fonts on your system
```
[the remark this answers is not in the document]
Comment (aweibel) [reply README.md]: Going directly to the font's page I linked to and downloading just the fonts is much easier than having to clone the repo and have a ton of other things that you don't need just to get the fonts. I thought I'd make it easier to find them. I really don't care too much either way. I can back this out if linking the fonts directly isn't ideal.

File: README.md (hunk 1 above)
Comment (aweibel) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (aweibel) [reply README.md]: Fixed

**Pull request merged**